pagopa/pn-frontend
# Rewroked "Altri recapiti" section - Feat/pn 12741

## Short description
Section "Altri recapiti" must be available only if at least one legal contact has been added.

## List of changes proposed in this pull request
- Updated Contacts.page 
- Updated SpecialContacts component
- Updated AddSpecialContactsDialog
- Updated SpecialContactItem
- Updated tests

## How to test
Login with pf/pg -> check that "Altri recapiti" section is shown only if at least one legal contact has been added -> check that only legal contacts can be added as contacts linked to a specific sender
